lib/util/api.ts

```diff
@@ -58,16 +59,16 @@ export function combineQueryParams(
   return qs.stringify(search, { arrayFormat: 'repeat' })
 }
 
-/**
- * Drops unused params so they don't show up in URL.
- * TODO: Remove dependency on getDefaultQuery from core-utils.
- */
-export function removeUnusedQueryParams(params: Record<string, any>): void {
-  delete params.showIntermediateStops
-  delete params.otherThanPreferredRoutesPenalty
-  delete params.ignoreRealtimeUpdates
-  delete params.optimize
-  delete params.optimizeBike
-  delete params.maxWalkDistance
-  delete params.maxBikeDistance
+/** Gets the default URL params when reinitializing search */
+export function getDefaultQuery() {
+  return {
+    date: getCurrentDate(),
+    departArrive: 'NOW',
+    // TODO: Rework the crash-related params below so we can remove them.
+    intermediatePlaces: [], // required to avoid crash
+    mode: 'WALK,TRANSIT', // obsolete but required to avoid crash
+    numItineraries: 3, // instead of 7 per apiV2 if no limit defined in config
```

Comment:
Won't this override the otp-rr default? Is this a required parameter?

Reply:
`numItineraries=3` shows up in the URL bar after a new search. If absent, many more results will be shown, especially on transit results.